sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/OptimizeInSuite.scala

```diff
@@ -128,4 +131,21 @@ class OptimizeInSuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
+  test("OptimizedIn test: Use configuration.") {
```

Comment:
I'd give this a more descriptive name, and explicitly say setting the threshold for turning into InSet